pallets/crowdloans/src/benchmarking.rs

```diff
@@ -202,9 +242,12 @@ benchmarks! {
     claim_refund {
         let ctoken = 15;
         let caller: T::AccountId = whitelisted_caller();
-        let crowdloan = ParaId::from(1341);
+        let crowdloan = ParaId::from(1341u32);
+        let cap = 1000_000_000_000_000;
```

Comment:
because nearly all crowdloan can share the same cap & end_block, you can define it once at the beginning